src/components/partners/index.js

```diff
@@ -53,27 +53,27 @@ const Parceires = () => {
           }
         }}
         modules={[Navigation]}
-        className="mySwiper"
+        className="my-swiper"
```

Comment:
Talvez `partner-swiper` seja mais descritivo?

```suggestion
        className="partner-swiper"
```